tce/src/applibs/LLVMBackend/LLVMBackend.cc

```diff
@@ -884,15 +884,10 @@ LLVMBackend::createPlugin(const TTAMachine::Machine& target) {
     }
 
     // NOTE: this could be get from Makefile.am
-    std::string pluginSources =
-        srcsPath + "TCERegisterInfo.cc " +
-        srcsPath + "TCEInstrInfo.cc " +
-        srcsPath + "TCEISelLowering.cc " +
-        srcsPath + "TCEDAGToDAGISel.cc " +
-        srcsPath + "TCETargetObjectFile.cc " +
-        srcsPath + "TCEFrameInfo.cc " +
-        srcsPath + "TCETargetMachinePlugin.cc " +
-        srcsPath + "TCESubtarget.cc ";
+    TCEString pluginSources = 
```

Comment:
This does not look correct. 4 spaces is the style guide. But in the original TCE style it would fill up the lines, for some reason your lines are not filled to 78 columns. Was this really what clang-format outputs?

Reply:
No I did that by hand to match the previous format and the format of some other strings a couple lines underneath it. I will run the format script and resubmit.